repo/migrations/Migration024_test.go

```diff
@@ -0,0 +1 @@
+package migrations_test
```

Comment:
I'd like to see a test that proves the table was created by adding and retrieving a full row with `Up()` and a that the table was removed on `Down()`.

Reply:
The test of adding a row, finding it is in repo/db/messages_test. The presence of the table after Up and its absence after Down have been tested here.